docker-compose-mysql-rabbitmq.yml

```diff
@@ -31,6 +31,11 @@ services:
     command: >
       sh -c "uwsgi --ini uwsgi.ini"
     environment: *oncall-environment
+    healthcheck:
+      test: ["CMD", "bash", "-c" , 'curl --fail http://127.0.0.1:8080/health/ || exit 1']
+      timeout: 5s
+      retries: 5
+      start_period: 20s
     depends_on:
```

Comment:
let's add the same dependencies [here](https://github.com/grafana/oncall/pull/1226/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R28) as well